recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -22,8 +22,8 @@ SRC_URI = " \
   file://aktualizr-secondary.socket \
   file://aktualizr-serialcan.service \
   "
-SRCREV = "930d8eef6eb584686654601c056d7c9c6fca3048"
-BRANCH ?= "master"
+SRCREV = "2b223de0ea262ae1a057a4eeaddd2314e3329c7a"
+BRANCH ?= "feat/garage-sign-auto-version"
```

Comment:
We probably don't want to merge the branch change.

Reply:
Agreed, still waiting on https://github.com/advancedtelematic/aktualizr/pull/684 to get merged first.